src/main.ts

```diff
@@ -197,7 +209,7 @@ async function run_linux(): Promise<void> {
 
     await core.group('Setting up repository', async () => {
       await exec.exec('sudo tee /etc/apt/sources.list.d/tarantool.list', [], {
-        input: Buffer.from(`deb ${baseUrl}/ubuntu/ ${distro} main\n`)
+        input: Buffer.from(`deb ${baseUrl}/${distro_id}/ ${distro} main\n`)
```

Comment:
I vote up this enhancement, but it doesn't look related to the pull request title: it is not about Ubuntu 22.04.

Since the pull request is more about docker container jobs rather than specific to Ubuntu 22.04 I would rephrase th pull request title/description.